biogeochem/EDPhysiologyMod.F90

```diff
@@ -1685,7 +944,8 @@ subroutine recruitment( currentSite, currentPatch, bc_in )
     real(r8) :: a_sapwood     ! sapwood cross section are [m2] (dummy)
     real(r8) :: b_agw         ! Above ground biomass [kgC]
     real(r8) :: b_bgw         ! Below ground biomass [kgC]
-
+    real(r8) :: b_dead
```

Comment:
it seems like you've made an effort to avoid the use of the word "dead" to refer to structural carbon in this parteh code, which I support since it can get confusing whether the dead relates to disturbance.  Should we try to be strict about this and replace the word "dead" when used in this sense with the word "structural" everywhere?

Reply:
I support that.  In our modeling meeting I think we agreed that we would slate that for a PR after the PARTEH v1.

Reply:
created an issue #430